### PR TITLE
Fix market hour pages headings

### LINKED ISSUE #1200: Inconsistent Market Hours Heading
#### Expected Behavior

All of the market hours pages have an H3 heading for **Regular Trading Hours**.

#### Actual Behavior

All of the asset classes use the **Regular Trading Hours** heading ([example](https://www.quantconnect.com/docs/v2/writing-algorithms/securities/asset-classes/us-equity/market-hours#03-Regular-Trading-Hours)), but the CFD and Futures market hours pages incorrectly use a [Market Opening Hours](https://www.quantconnect.com/docs/v2/writing-algorithms/securities/asset-classes/cfd/market-hours/au200aud#03-Market-Opening-Hours) heading.

#### Checklist

- [x] I have completely filled out this template
- [x] I have confirmed that this issue exists on the current `master` branch
- [x] I have confirmed that this is not a duplicate issue by searching [issues](https://github.com/QuantConnect/Lean/issues)

### PR DESCRIPTION
**Pull request by LouisSzeto**
#### Description
Correction in CFD/Futures generated pages' heading from "Market Opening Hours" -> "Regular Trading Hours"

#### Related Issue
closes #1200

#### Motivation and Context
Consistency

#### Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which improves implementation)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Non-functional change (xml comments/documentation/etc)

#### Checklist:

- [X] My code follows the code style of this project.
- [X] I have read the **CONTRIBUTING** [document](https://github.com/QuantConnect/Lean/blob/master/CONTRIBUTING.md).
- [X] My branch follows the naming convention `bug-<issue#>-<description>` or `feature-<issue#>-<description>`

### CODE REVIEW

File: Resources/datasets/market-hours/equity/india/generic/regular-trading-hours.html
Comment (DerekMelchin) [inline Resources/datasets/market-hours/equity/india/generic/regular-trading-hours.html:1]: If you rename this file, you also need to adjust the `include` statement in `Documentation\03 Writing Algorithms\03 Securities\99 Asset Classes\02 India Equity\05 Market Hours\03 Regular Trading Hours.php`

File: Resources/datasets/market-hours/equity/usa/generic/regular-trading-hours.html
Comment (DerekMelchin) [inline Resources/datasets/market-hours/equity/usa/generic/regular-trading-hours.html:1]: If you rename this file, you also need to adjust the `include` statement in `Documentation\03 Writing Algorithms\03 Securities\99 Asset Classes\01 US Equity\09 Market Hours\03 Regular Trading Hours.php`

File: Resources/datasets/market-hours/index/usa/generic/regular-trading-hours.html
Comment (DerekMelchin) [inline Resources/datasets/market-hours/index/usa/generic/regular-trading-hours.html:1]: If you rename this file, you also need to adjust the `include` statement in `Documentation\03 Writing Algorithms\03 Securities\99 Asset Classes\09 Index\04 Market Hours\03 Regular Trading Hours.php`

File: Resources/datasets/market-hours/option/usa/generic/regular-trading-hours.html
Comment (DerekMelchin) [inline Resources/datasets/market-hours/option/usa/generic/regular-trading-hours.html:1]: If you rename this file, you also need to adjust the `include` statement in `Documentation\03 Writing Algorithms\03 Securities\99 Asset Classes\03 Equity Options\04 Market Hours\03 Regular Trading Hours.php`

**Review by DerekMelchin — CHANGES_REQUESTED** [review]

**Review by AlexCatarino — CHANGES_REQUESTED** [review]
We should review the possible improvement of this code generation. It seems odd that we have a market-hour-convertor and a market-hour-code-generator. Why the convertor is not part of the generator? We had to replace "market-opening-hours.html" with "regular-trading-hours.html" which shows that these scripts are error-prone.

I have created the `_code_generation_helpers.py` to avoid code duplication. We can use it to define global variables.

**Comment by LouisSzeto** [comment]
> We should review the possible improvement of this code generation. It seems odd that we have a market-hour-convertor and a market-hour-code-generator. Why the convertor is not part of the generator?

This is mainly due to the workflow of `source -> RESOURCES -> production page`, so we separated 2 files here.

**Pull request merged**